spec/telephone_appointments/summary_document_activity_spec.rb

```diff
@@ -1,7 +1,24 @@
 require 'spec_helper'
 
 RSpec.describe TelephoneAppointments::SummaryDocumentActivity do
-  subject { described_class.new(appointment_id: 12, owner_uid: 'abc') }
+  subject { described_class.new(appointment_id: 12, owner_uid: 'abc', delivery_method: 'postal') }
+
+  describe '.new' do
+    context 'with a valid delivery method' do
+      it 'does not raise an error' do
+        expect { described_class.new(appointment_id: 1, owner_uid: '', delivery_method: 'postal') }.not_to raise_error
```

Comment:
Just calling these initializers is enough to warrant not throwing an error. I'd say remove the `expect` cruft and you won't lose any clarity here.